backend/server/swagger_ui.py

```diff
@@ -0,0 +1,10 @@
+from flask_swagger_ui import get_swaggerui_blueprint
+
+
+SWAGGER_URL = '/api/docs'
+API_URL = 'https://raw.githubusercontent.com/meetingbook/meetingbook/main/backend/api/meetingbook.yaml'
```

Comment:
I think it would be better to use a path from the current repository. 
For example, if I change the yaml file I want to see the changes already in swagger UI even before I commit it.

Reply:
![image](https://user-images.githubusercontent.com/83371532/131458537-4ee96160-1d7d-4d6d-bec8-4b4bab2e4a95.png)

hint: https://stackoverflow.com/questions/55733136/flask-swagger-ui-does-not-recognize-path-to-swagger-json

Reply:
@mentortechabc I wanted to do it with a local file, but yesterday it didn't work for me. Thanks for the hint, now everything worked out😀